### PR TITLE
Add tests for UI: reducer, selector and type

### PR DESCRIPTION
**Pull request by mitogh**
**Note** This PR requires: https://github.com/moderntribe/events-gutenberg/pull/166

### CODE REVIEW

File: src/modules/data/search/__tests__/actions.test.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+describe( '[STORE] - Search actions', () => {
+	it( 'Should add a block', () => {
+		const expectedAction = {
+			type: types.ADD_BLOCK,
+			payload: {
+				id: 1,
+			},
+		};
+		expect( actions.addBlock( 1 ) ).toEqual( expectedAction );
```
Comment (nealfennimore) [inline src/modules/data/search/__tests__/actions.test.js]: Actions are a great case for using snapshots. I'd recommend doing that with other actions so you don't have to do so much setup work. All you'd need to do is:

```js
expect( actions.addBlock( 1 ) ).toMatchSnapshot();
```

Just make sure you verify the output is correct :)

File: src/modules/data/ui/__tests__/reducer.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+/**
+ * Internal dependencies
+ */
+import reducer, { DEFAULT_STATE } from 'data/ui/reducer';
+import { actions } from 'data/ui';
+
+describe( '[STORE] - UI reducer', () => {
+	it( 'Should return the default state', () => {
+		const current = reducer( undefined, {} );
+		expect( current ).toEqual( DEFAULT_STATE );
```
Comment (nealfennimore) [inline src/modules/data/ui/__tests__/reducer.test.js]: Snapshots are good for reducer output as well :)

**Comment by mitogh** [comment]
Thanks @nealfennimore 

Would you mind take a look at: 

- https://github.com/moderntribe/events-gutenberg/pull/166

As is one before this one and will update this one with some snapshots, thanks for the recommendation.

**Comment by mitogh** [comment]
Another look on this one would be great. @nealfennimore 

Thanks for the recommendation makes test much faster to write 💃

**Pull request merged**